tls/README.md

```diff
@@ -0,0 +1,48 @@
+# Agent Check: TLS
+
+## Overview
+
+This check monitors [TLS][1] protocol versions, certificate expiration & validity, etc.
+
+## Setup
+
+### Installation
+
+The TLS check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `tls.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your TLS data. See the [sample tls.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3].
+
+### Validation
+
+[Run the Agent's status subcommand][4] and look for `tls` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][5] for a list of metrics provided by this integration.
+
+### Service Checks
+
+See [service_checks.json][6] for a list of service checks provided by this integration.
```

Comment:
The service checks need to be listed out for the docs. For example:

```
tls.can_connect  
Returns CRITICAL if the Agent is unable to connect to the monitored endpoint, otherwise returns OK.
```